src/Http/Livewire/LivewireDatatable.php

```diff
@@ -698,7 +698,13 @@ public function addGlobalSearch()
                     $this->searchableColumns()->each(function ($column, $i) use ($query, $search) {
                         $query->orWhere(function ($query) use ($i, $search) {
                             foreach ($this->getColumnField($i) as $column) {
-                                $query->orWhereRaw('LOWER('.$column.') like ?', "%$search%");
+                                if (is_array($column)) {
+                                    foreach ($column as $col) {
+                                        $query->orWhereRaw("LOWER(" . $col . ") like ?", "%$search%");
```

Comment:
I added a suggestion that needs to be added to both `$query` parts to get the build green.

```suggestion
$query->orWhereRaw('LOWER('.$column.') like ?', "%$search%");
```